app/Http/Middleware/SuperAdminOnly.php

```
@@ -0,0 +1,23 @@
+use Symfony\Component\HttpFoundation\Response;
+
+class SuperAdminOnly
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param Closure(Request): (Response) $next
+     */
+    public function handle(Request $request, Closure $next): Response
+    {
+        if ($request->user()?->isSuperAdmin()) {
```

PHPStan (2.0.3)

Call to an undefined method App\Models\User::isSuperAdmin().


(method.notFound)